src/ui/screens/home/controller/ControllerView.qml

```diff
@@ -141,6 +141,46 @@ Item {
                 visible: false
             }
 
+        },
+        State {
+            name: "stateRegeneratingKey"
+            when: (VPNController.state === VPNController.StateRegeneratingKey)
+
+            PropertyChanges {
+                target: boxBackground
+                color: MZTheme.colors.primary
+            }
+
+            PropertyChanges {
+                target: logoTitle
+                //% "Connecting…"
```

Comment:
This comment tells the qml-translation pipeline what "real" string we want to have for that, we only need that type of comment once, so it's not needed here :)